.github/workflows/continuous-build.yml

```diff
@@ -23,15 +23,12 @@ jobs:
     strategy:
       matrix:
         node-version: [18, 20, latest]
-        pnpm-version: [9]
```

Comment:
I didn't see the need to have the pnpm version in the matrix strategy, particularly since we only use v9, so I removed it. This will also avoid conflicts with the Action saying that the Action pnpm version doesn't match the `packageManager` pnpm version ([new in v4](https://github.com/pnpm/action-setup/releases/tag/v4.0.0)).